package.json

```diff
@@ -94,10 +94,11 @@
     "format-ci": "prettier --list-different '**/*.{js,css}'",
     "format-docs": "prettier --write '**/*.md'",
     "test": "NODE_ICU_DATA=node_modules/full-icu sharetribe-scripts test",
-    "test-ci": "sharetribe-scripts test --runInBand",
+    "test-ci": "yarn run test-server && sharetribe-scripts test --runInBand",
     "eject": "sharetribe-scripts eject",
     "start": "node --icu-data-dir=node_modules/full-icu server/index.js",
     "dev-server": "export NODE_ENV=development PORT=4000 REACT_APP_CANONICAL_ROOT_URL=http://localhost:4000&&yarn run build&&nodemon --watch server server/index.js",
+    "test-server": "jest ./server/**/*.test.js",
```

Comment:
CRA only supports automatically running tests inside `/src` so we need to add another script for server side tests. See e.g. https://spectrum.chat/create-react-app/general/how-can-tests-outside-of-src-be-included-in-the-test-run~3610ecf1-a396-466b-8f62-a62627062151

Reply:
You probably should add the [--runInBand](https://jestjs.io/docs/en/cli.html#--runinband) option also for the server Jest command. Circle CI at least used to have lots of performance problems without that option.